README.md

```diff
@@ -1,4 +1,4 @@
-<img align="right" src="https://cdn.miranj.in/mc3/img/apple-touch-icon-precomposed.png" width="57" height="57" alt="Miranj">
+<img align="right" src="https://cdn.miranj.in/v4/favicons/apple-touch-icon.png" width="57" height="57" alt="Miranj">
```

Comment:
Incorrect logo version. Use the version that is used in, say, our Craft Partners profile. https://craftcms.com/partners/miranj

Reply:
Updated from the Craft Partners Profile. 

Note: The image is square but on the Craft site the styles make it rounded.